docs/releases/v1.6.13.md

```diff
@@ -0,0 +1,28 @@
+# Release v1.6.13 (8th April 2022)
+
+The release is called **v1.6.13**, an update to support version 3.1.10 of the AIS, PIS, CBPII and VRP Open Banking APIs.
+
+Issues addressed:
```

Comment:
Check if the issues are really FCS specific, or just my mistake from before.
If not strictly FCS related - remove.